pxl_scripts/px/redis_data/manifest.yaml

```diff
@@ -0,0 +1,6 @@
+---
+short: Redis RPC messages
+long: >
+  Traces Redis RPC request and response messages.
+  WARNING: Redis tracing is a beta feature. Details may subject to further changes.
```

Comment:
I think the warning/note makes sense when loading the table inside the script (which you already have).

Here, maybe just say: "Redis tracing is a beta feature. This script is for demonstration purposes only."